config/default.json

```diff
@@ -46,11 +51,12 @@
   },
   "tracker": {
     "githubIssues": {
-      "repository": "OpenTermsArchive/sandbox",
+      "repository": "ambanum/test-repo",
```

Comment:
Because today, the repo `OpenTermsArchive/sandbox` does not exist and I need one when I want to test if the tracker works properly. Ideally we should create this repo and update rights and tokens accordingly.